packages/graph-explorer/src/modules/ConnectionDetail/ConnectionDetail.tsx

```diff
@@ -37,6 +37,12 @@ export type ConnectionDetailProps = {
 };
 
 const HEADER_ACTIONS = (isSync: boolean, isFileBased: boolean) => [
+  {
+    label: "Synchronize Database",
```

Comment:
@cubeddu, what would be the strategy to support i18n?

Reply:
Not currently, I when ahead and added a feature request here issue #128